internal/nodes/avi_model_evh_nodes.go

```diff
@@ -0,0 +1,1029 @@
+package nodes
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"regexp"
+	"sort"
+	"strings"
+
+	avicache "github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/internal/cache"
+	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/internal/lib"
+	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/internal/objects"
+	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/utils"
+
+	avimodels "github.com/avinetworks/sdk/go/models"
+	networkingv1beta1 "k8s.io/api/networking/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type AviEvhVsNode struct {
+	EVHParent        bool
+	VHParentName     string
+	VHDomainNames    []string
+	EvhNodes         []*AviEvhVsNode
+	EvhHostName      string
+	EvhPath          string
+	EvhMatchCriteria string
+	// props from avi vs node
+	Name                  string
+	Tenant                string
+	ServiceEngineGroup    string
+	ApplicationProfile    string
+	NetworkProfile        string
+	Enabled               *bool
+	PortProto             []AviPortHostProtocol // for listeners
+	DefaultPool           string
+	EastWest              bool
```

Comment:
Is this needed?

Reply:
Had retained all of the props initially as I was thinking of embedding the struct and reusing the functions. Not needed now. Removed all the references of the properties and their refs which are not used.